Configure

```diff
@@ -2325,6 +2325,42 @@ EOF
             delete $unified_info{includes}->{$dest};
         }
     }
+
+    # For convenience collect information regarding directories where
+    # files are generated, those generated files and the end product
+    # they end up in where applicable.  Then, add build rules for those
+    # directories
```

Comment:
I'm not sure I understand the purpose of moving this code?

Reply:
To make it visible on `configdata.pm`. It's quite hard to see the results otherwise.